psi4/driver/driver_cbs.py

```diff
@@ -1122,25 +1635,37 @@ def cbs(func, label, **kwargs):
 
        - Need to add more extrapolation schemes
 
-    As represented in the equation below, a CBS energy method is defined in several
-    sequential stages (scf, corl, delta1, delta2, ... ) covering treatment
-    of the reference total energy, the correlation energy, a delta correction to the
-    correlation energy, and a second delta correction, etc.. Each is activated by its
-    stage_wfn keyword, or as a field in the ```cbs_metadata``` list, and is only
-    allowed if all preceding stages are active.
+    As represented in the equation below, a CBS energy method is defined in 
+    several sequential stages (scf, corl, delta1, delta2, ... ) covering treatment
+    of the reference total energy, the correlation energy, a delta correction to 
+    the correlation energy, and a second delta correction, etc. Each is activated 
+    by its stage_wfn keyword, or as a field in the ```cbs_metadata``` list, and 
+    is only allowed if all preceding stages are active.
+
+    The extrapolation of DFT calculations from basis sets of successive zeta 
+    levels is also implemented. By default, the components of the DFA recipe
+    (fctl, dh, disp, nl) are treated separately: the fctl component (equivalent
+    to the scf stage in WFT) is extrapolated using the exponential -- square root
+    function; the dh component (equivalent to the corl stage in WFT) is
+    extrapolated using the power function; and the disp and nl components are not
+    extrapolated at all.
 
     .. include:: /cbs_eqn.rst
 
-    * Energy Methods
-        The presence of a stage_wfn keyword is the indicator to incorporate
-        (and check for stage_basis and stage_scheme keywords) and compute
-        that stage in defining the CBS energy.
+    .. _cbs-energy-methods:
+
+    Energy Methods
+    --------------
+    
+    The presence of a stage_wfn keyword is the indicator to incorporate (and check
```

Comment:
Presence of a `stage_wfn` keyword _in what_?

This documentation reads to me like `stage_wfn` is a boolean that controls whether or not the stage should be part of the CBS extrapolation. What am I missing?

Reply:
No, stage_wfn are arguments that you pass to cbs.